src/pudl/transform/eia860.py

```diff
@@ -78,6 +83,18 @@ def generators(eia860_dfs, eia860_transformed_dfs):
     the record came from. We use ``operational_status`` to parse the pre 2009
     files as well.
 
+    Transformations include:
+    - Replace . values with NA.
```

Comment:
Needs a blank line between list and preceding text -- as do all of the other bulletized lists.